changelog_entry.yaml

```diff
@@ -0,0 +1,5 @@
+- bump: minor
+  changes:
+    added:  
+    - NYSERDA Drive Clean program.
+  date: 2024-07-01 00:00:00
```

Comment:
```suggestion
```

Reply:
The date will be auto populated when the PR is merged, no need to manually enter it

Reply:
Oh! Sorry, I don't think I understand. Should I delete the date and leave the field blank.